### PR TITLE
Improve spelling

### PR DESCRIPTION
**Pull request by LEJOUI**
## Proposed change

The `...` of this line keeps bugging me. It looks unfinished. I'd like to change it in a line that is at least complete. 
Here's my proposal. Feedback is appreciated.
## Type of change

- [ ] Dependency upgrade
- [ ] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (thank you!)
- [ ] Breaking change (fix/feature causing existing functionality to break)
- [ ] Code quality improvements to existing code or addition of tests

## Example configuration

```yaml

```

## Additional information

- This PR fixes or closes issue: fixes #
- This PR is related to issue or discussion:
- Link to documentation pull request:

## Checklist

- [ ] The code change is tested and works locally.
- [ ] There is no commented out code in this PR.
- [ ] Tests have been added to verify that the new code works.

If user exposed functionality or configuration variables are added/changed:

- [ ] Documentation added/updated for [www.home-assistant.io][docs-repository]

[docs-repository]: https://github.com/home-assistant/home-assistant.io

### CODE REVIEW

File: src/translations/en.json [hunk 1]
```diff
@@ -2103,7 +2103,7 @@
         },
         "integrations": {
           "caption": "Integrations",
-          "description": "Manage integrations with services, devices, ...",
+          "description": "Manage integrations to add services, devices and entities",
```
Comment (frenck) [inline src/translations/en.json:2106]: - It's not just adding.
- It manages integrations that interact with devices or services. Entities can be a result of that, but it not what is meant here.

**Comment by LEJOUI** [comment]
Thanks for your feedback!
I'm also thinking that one of these two may fit

1. Manage integrations with services or devices
2. Manage integrations that link with services or devices

**Comment by spacegaier** [comment]
"Manage integration **that**" always sounds as if only a subset of all integrations are in there, when if fact all are in there.

=> I would vote for alternative 1: Manage integrations with services or devices (could also be "and")

**Comment by frenck** [comment]
It is `or` an integration can only expose a single type; as in either device or service.

**Review by spacegaier — APPROVED** [review]

**Pull request merged**